src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceKeysetsDisabledTest.java

```diff
@@ -0,0 +1,660 @@
+package com.uid2.admin.vertx;
+
+import com.uid2.admin.store.Clock;
+import com.uid2.admin.vertx.service.EncryptionKeyService;
+import com.uid2.admin.vertx.service.IService;
+import com.uid2.admin.vertx.test.ServiceTestBase;
+import com.uid2.shared.auth.Keyset;
+import com.uid2.shared.auth.Role;
+import com.uid2.shared.model.EncryptionKey;
+import com.uid2.shared.model.KeysetKey;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.HttpResponse;
+import io.vertx.junit5.VertxTestContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+public class EncryptionKeyServiceKeysetsDisabledTest extends ServiceTestBase {
```

Comment:
- I guess all these copied tests are ok since it's temporary code which we'll remove in a couple months?
Otherwise we should use a technique like EUIDOperatorVerticleTest where the entire test suite is run again but with minor diffs for EUID.
- Are there any tests where we confirm the files are not created when they otherwise would be? Can we do this after the end of every test in this file, eg with some JUnit technique?

Reply:
I thought about making it parameterized, but I think for this being temporary and a test, its less effort and will be removed.

Added
```java
verifyNoInteractions(keysetStoreWriter);
verifyNoInteractions(keysetKeyStoreWriter);
```
To the end of every test in the disabled one.

Reply:
let's use `@AfterEach` instead if that works